make_hh_acs_data.R

```diff
@@ -83,11 +84,17 @@ get_acs_poverty <- function(year) {
     mutate(year = year)
 }
 
-d_acs$acs_poverty <-
-  mappp_dfr(2010:2021, get_acs_poverty) |>
-  add_col_attrs(fraction_poverty,
-    title = "Fraction of Households in Poverty",
-    description = "Fraction of households with income below poverty level within the past 12 months"
+d_acs$acs_poverty <- mappp_dfr(2010:2022, get_acs_poverty) |>
+  as_fr_tdr(
+    name = "hh_acs_measures",
+    version = "1.2.0",
+    title = "Harmonized Historical American Community Survey Measures",
+    description = "2010 - 2022 measures derived from ACS variables for census tracts in the contiguous US",
+    homepage = "https://geomarker.io/hh_acs_measures"
+  ) |>
+  update_field("fraction_poverty",
+               title = "Fraction of Households in Poverty",
+               description = "Fraction of households with income below poverty level within the past 12 months"
```

Comment:
note: to retain overall tdr metadata, I think we have to assign it to first tdr in the list... a limitation of the way I have currently written `fr_left_join()`

Reply:
doing it at the end as we were doing previously overwrites all the other metadata